store-implementation/hbase-store/src/main/java/uk/gov/gchq/gaffer/hbasestore/serialisation/ElementSerialisation.java

```diff
@@ -83,13 +83,19 @@ public ElementSerialisation(final Schema schema) {
                 try {
                     if (null != serialiser) {
                         Object value = properties.get(propertyName);
+                        byte[] bytes;
                         if (null != value) {
-                            final byte[] bytes = serialiser.serialise(value);
-                            writeBytes(bytes, out);
+                            bytes = serialiser.serialise(value);
+                        } else if (null != this.schema.getVisibilityProperty()) {
```

Comment:
nonNull()

Reply:
This isn't uniform across the codebase and is preference, I personally find `null != x` easier to read.

Reply:
:D when one is seen it gets changed. This is best way to avoid a ticket to change them all. :D
I request this in all PR.
nonNull is 100% more human readable :D it has words in it for a start.

Reply:
Is `!=` reinventing the wheel? I would argue `nonNull` is

Reply:
Your not implementing it and its part of core Java API. so you'd be"making use of core Java API"
but if you feel really strongly to not have it.

> "API Note: This method exists to be used as a Predicate, filter(Objects::isNull)"

Reply:
Yeah, the method was made to make predicates nicer than `filter(x -> x == null)`. The implementation is just
```java
 public static boolean isNull(Object obj) {
     return obj == null;
 }
```
Personally it is not nicer to read so I think it is just a preference thing.